charts/hyades/values.yaml

```diff
@@ -204,6 +205,7 @@ frontend:
   service:
     type: ClusterIP
     nodePort: ~
+    port: 8080
```

Comment:
Here, I kept what is on the original code. But it should be 8081. Refer to: https://github.com/DependencyTrack/helm-charts/pull/157